website/src/content/development/adding-typescript.mdx

```diff
@@ -0,0 +1,101 @@
+---
+title: Adding TypeScript
+---
+
+[TypeScript](https://www.typescriptlang.org/) is a typed superset of JavaScript that compiles to plain JavaScript.
+
+Custom Applications have full support for developing applications in TypeScript.
+
+<Info>
+
+This feature is available from version `>= 21.8.0`.
```

Comment:
```suggestion
This feature is available from version `21.8.0`.
```

Using the word "from" should imply greater than or equal to the stated version.

Reply:
Thanks for the suggestion. If we change this, there are also other places in the docs that would need to be updated.

Can you confirm that it's clear that the "equal" part is implied? Not every user is a native english speaker, not I'm not sure if that's really obvious... 🤔

To be on the safe side, would it still be ok with you if we keep the `>=`?

Reply:
Sure, you could keep `>=` for now, or you could consider the following as well:

```suggestion
This feature is available from version `21.8.0` onwards.
```

Reply:
Thanks, I'll use the `onwards` version.